conda-forge.yml

```diff
@@ -4,3 +4,5 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+noarch_platforms:
+  - win_64
```

Comment:
We need to manually add this bit, right? It would be better if we could do everything from the recipe but I'm not familiar with all the details of this implementation to help.

Anyway,. LGTM. Let's test it out!

Reply:
Yeah I think so. If someone thinks of a better way to do this, agree that would be great!

Thanks Filipe 😄

Reply:
Ideally this should be at a lower level, in conda itself, but that would be an herculean effort.